modules/ocf_irc/files/anope/chanserv.conf

```diff
@@ -0,0 +1,484 @@
+# Comments trimmed for easier editing.
```

Comment:
I definitely agree with your feeling about this, since these are really large config files. I tried to use augeas before, but discovered there is no lens for this format, which really sucks. Sed would work fine, but it's at that awkward point where enough has been changed that a lot of sed commands would need to be written to get the same effect, but that might still be a better option than committing the whole thing.

Includes would be best, not sure if that would work, I hadn't considered it. If later options overwrite earlier config options (or vice-versa), then that would work fine, but I'm not sure how anope behaves on this.